core/io/pck_packer.cpp

```diff
@@ -72,7 +72,12 @@ Error PCKPacker::pck_start(const String &p_file, int p_alignment) {
 	file->store_32(1); // # version
 	file->store_32(VERSION_MAJOR); // # major
 	file->store_32(VERSION_MINOR); // # minor
-	file->store_32(0); // # revision
+#ifdef VERSION_PATCH
+	file->store_32(VERSION_PATCH); // # revision
```

Comment:
The comment should say "patch", not "revision" (which used to be what we now call "build_name", i.e. "custom_build" or "official" or anything specified with `BUILD_NAME` when compiling).